kubeprod/pkg/installer/install.go

```diff
@@ -122,14 +122,14 @@ func (c InstallCmd) Run(out io.Writer) error {
 
 	// TODO(felipe): Conditionalize this with a command-line flag so this
 	// step is optional
-	if true {
+	if c.OnlyGenerate {
+		fmt.Println("Kubernetes cluster is ready for deployment.")
+		fmt.Println("run: `kubeprod install` without the --only-generate command-line flag.")
```

Comment:
This last line is slightly ambiguous - is it saying that's what is happening?  Does the user always have to run that next command line now? etc.

Maybe just state the simple truth: `"Exiting early because --only-generate was provided."`  (Or perhaps `"Skipping deployment because ..."`)